platform/broadcom/saibcm-modules/make/Makefile.linux-x86-smp_generic_64-2_6

```diff
@@ -28,18 +28,29 @@ ifeq (,$(KFLAGS))
 KFLAGS := -nostdinc -isystem $(SYSINC) -I$(KERNDIR)/include -I$(KERNDIR)/arch/x86/include -include $(AUTOCONF) -D__KERNEL__ -Wall -Wundef -Wstrict-prototypes -Wno-trigraphs -fno-strict-aliasing -fno-common -Werror-implicit-function-declaration -Wno-format-security -fno-delete-null-pointer-checks -Os -m64 -mtune=generic -mno-red-zone -mcmodel=kernel -funit-at-a-time -maccumulate-outgoing-args -DCONFIG_AS_CFI=1 -DCONFIG_AS_CFI_SIGNAL_FRAME=1 -pipe -Wno-sign-compare -fno-asynchronous-unwind-tables -mno-sse -mno-mmx -mno-sse2 -mno-3dnow -fno-stack-protector -fomit-frame-pointer -g -Wdeclaration-after-statement -Wno-pointer-sign
 endif
 
-ifeq ($(LINUX_MAKE_SHARED_LIB), 1)
-KFLAGS += -fPIC -mcmodel=small
-else
-KFLAGS += -fno-pie -mcmodel=kernel
```

Comment:
`-fno-pie `was previously added for a compilation issue `gmodule.c:1:0: error: code model kernel does not support PIC mode`